changelogs/unreleased/3950-compile-report-attribute.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: add indication for "Compile Reports" navigation link that inform about any ongoing compilations
```

Comment:
Add an indication in the navigation sidebar when there are ongoing compilations. The indication will be added to the "Compile Report" menu-item.